.github/workflows/dotnet-desktop.yml

```diff
@@ -0,0 +1,79 @@
+
+name: BuildMain
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches: [ main ]
+env:
+        configuration: Release
+        productNamespacePrefix: "ReactiveUI"
+        
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: [Release]
```

Comment:
You can use the matrix for the solutions too.

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix